.github/workflows/build.yml

```diff
@@ -205,6 +205,61 @@ jobs:
         run: |
           cat ./build/meson-logs/testlog.txt
 
+  linux-ffmpeg-4-4:
+    runs-on: ubuntu-22.04
+    container:
+      image: alpine:3.15
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          apk update
+          apk add \
+              binutils \
+              ffmpeg-dev \
+              gcc \
+              git \
+              g++ \
+              glslang-dev \
```

Comment:
it *could* be interesting to do a more minimal build without vulkan, lcms or vaapi. in case anyone every accidentally breaks that.

Reply:
also on this line in particular:
>  libplacebo| Message: Skipping `glslang` because `shaderc` is available

Reply:
So basically remove vulkan headers, lcms, and libva?

Reply:
yeah, also shaderc